src/app/beer_garden/config.py

```diff
@@ -32,12 +32,16 @@ def load(args: List[str], force=False):
 
     Once loaded, the configuration can be reached with the `get` method.
 
-    :param args: Command line arguments.
-    :param force: Force a reload.
+    Args:
+        args: Command line arguments
+        force: Force a reload
+
+    Returns:
+        The configuration object
```

Comment:
Conceptually, I don't think we should be returning the configuration object here. The intent is always to access the config object through `config.get`. This opens us up to someone modifying the configuration object outside of the scope of the config subsystem.

Reply:
:+1: